src/components/ThreejsEditorModal.js

```diff
@@ -108,6 +108,14 @@ export class ThreejsEditorModal extends ModalDialog {
         const loader = new THREE.ObjectLoader();
         const scene = loader.parse(materialsToThreeDSceneData(this.props.materials));
         this.editor.execute(new window.SetSceneCommand(scene));
+
+        // Add a light on the other side
+        const directionalLight = new THREE.DirectionalLight("#FFFFFF");
+        directionalLight.name = "DirectionalLight_2";
+        directionalLight.position.copy(new THREE.Vector3(50, 0, 10));
```

Comment:
How did we come up with the coordinates 50, 0, 10?? It should be explained in a comment and/or the coordinates should be called a variable (e.g. `secondDirectionalLightPosition`) and imported from the settings file.